### DISCUSSION TITLE
[求助] 有异性朋友来家里吃饭该做什么菜招待呢？

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#1598] [其他]
![IMG_20240812_101724](https://github.com/user-attachments/assets/14dd5d44-ff4f-4b4e-a7ca-1ef27166c2e1)

> 今天和妈妈打电话的时候偶遇了录像店店长，不料妈妈居然从电话那边也听到了店长的声音。让我替她向店长问好，还说下班之后邀请他到家里做客……话说就算是朋友也不该这么热情吧…

![IMG_20240812_101710](https://github.com/user-attachments/assets/8530a9c1-e340-4d1c-a5b8-4feb62bb7dfa)

> 姑且是先答应了，店长好像也很感兴趣，我说了会做几道家常菜，但实际上完全没想好要做什么（店长会爱吃番茄炒蛋么？）

**Comment by xtd114514** [comment]
建议直接西红柿炒番茄，土豆炒马铃薯

**Comment by sishui-buyin** [comment]
建议做巧克力凉拌可可豆

**Comment by SSTV-10068** [comment]
听说六分街的"CAFF COFE"推出了新的产品，可以带回家一起品尝喔~
